src/refg.cpp

```diff
@@ -68,9 +69,52 @@ refg_t refg_registry_t::as_refg(std::string_view config) const
 	return ref;
 }
 
+std::string_view refg_registry_t::_try_refg_as_sv_from_file(refg_t ref) const
+{
+	std::string path{fmt::format("{}.hash", ref)};
+	if (!std::filesystem::exists(path)) {
+		path = resolve_datafile_path(prepend_dir(data_dir(), path));
+	}
+	if (!std::filesystem::exists(path)) {
+		return {};
```

Comment:
this is returning pointers to local variables (that will be delteted).

change the fn signature to return a `std::string` instead

Reply:
actually I misread this (long day).  default ctor on a string_view is fine since it doesn't have a ptr to anything.